frontend/app/utils/logging.server.ts

```diff
@@ -1,12 +1,18 @@
-import { type Logger, createLogger, format, transports } from 'winston';
+import winston, { type Logger, createLogger, format, transports } from 'winston';
+import { z } from 'zod';
 
-import { getEnv } from './env.server';
+// ['error', 'warn', 'info', 'verbose', 'debug', 'silly']
+const logLevels = Object.keys(winston.config.npm.levels);
 
-export const getLogger = (category: string): Logger => {
-  const { LOG_LEVEL } = getEnv();
+const logLevel = z
+  .string()
+  .refine((val) => logLevels.includes(val), { message: 'Invalid log level', path: ['process.env.LOG_LEVEL'] })
+  .default('info')
+  .parse(process.env.LOG_LEVEL);
```

Comment:
Is it working of the logger is initialised on the client since it doesn't have access to `process.env.LOG_LEVEL`?

Reply:
Will approve since `getLogger` is only called in server files for now. Keep in mind that we might want to log on the client later. :)